hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -302,3 +316,93 @@ func MachineTemplateSpec(nodePool *hyperv1.NodePool, bootImage BootImage, hclust
 		},
 	}
 }
+
+func ReconcileIgnitionWorkerConfigMap(cm *corev1.ConfigMap, nodePool *hyperv1.NodePool) error {
+	machineConfig := WorkerMachineConfig()
+	serializedConfig, err := workerDualSackConfig()
+	if err != nil {
+		return fmt.Errorf("failed to serialize ignition config: %w", err)
+	}
+	machineConfig.Spec.Config.Raw = serializedConfig
+	return reconcileMachineConfigIgnitionConfigMap(cm, machineConfig, config.OwnerRefFrom(nodePool))
+
+	return nil
+}
+
+func workerDualSackConfig() ([]byte, error) {
+	configureDHCP := `
+interfaces:
+- name: enp1s0
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: true
+  ipv6:
```

Comment:
why is this hypershift specific? Why is not coming from the MCO rendering?

Reply:
We are checking what alternatives we have.

Reply:
@enxebre something like this https://github.com/openshift/machine-config-operator/pull/3780 make more sense right ?

Reply:
yep, we're going to try and get this into the MCO, that was my recommendation as well after looking at this.